app/src/main/res/layout/bottom_right_overlay.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+    <variable
+      name="listener"
+      type="org.oppia.android.app.spotlight.SpotlightNavigationListener" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/spotlight_overlay_constraint_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/transparent_background"
+    android:clickable="true"
+    android:focusable="true">
+
+    <ImageView
+      android:id="@+id/spotlight_overlay_arrow"
+      android:layout_width="@dimen/arrow_width"
```

Comment:
Don't use `Fixed Width` and `FixedHeight` for textviews. As suggested in one of above comments.

```
layout_width="wrap_content"
layout_height="wrap_content"
minWidth="160dp"
minHeight="48dp"
```

Reply:
done

Reply:
I believe this is either meant to be for the close_target button or this is outdated due to my recent code push. Changes are addressed for the text view which is now changed to a button.